src/js/jobs/components/JobArgs.tsx

```diff
@@ -0,0 +1,191 @@
+ * @param className - An optional class name to apply to the row
+ * @returns A table row containing the argument name and value
+ */
+function JobArgsRow({ children, title, className }: { children: ReactNode; title: string; className?: string }) {
+    return (
+        <tr className={className}>
+            <th>{title}</th>
+            <td>{children}</td>
+        </tr>
+    );
+}
+
+/**
+ * Rows showing important arguments when running a sample analysis workflow.
+ *
+ * @param args - The complete list of arguments used to run the job
+ * @returns Table rows containing the arguments of the job
+ */
+export function AnalysisRows({ sample_id, analysis_id }: { sample_id: string; analysis_id: string }) {
+    return (
+        <>
+            <JobArgsRow title="Sample">
+                <Link to={`/samples/${sample_id}`}>{sample_id}</Link>
+            </JobArgsRow>
+            <JobArgsRow title="Analysis">
+                <Link to={`/samples/${sample_id}/analyses/${analysis_id}`}>{analysis_id}</Link>
+            </JobArgsRow>
+        </>
+    );
+}
+
+/**
+ * Rows showing important arguments when running an "build_index" workflow.
+ *
+ * @param args - The complete list of arguments used to run the job
+ * @returns Table rows containing the arguments of the job
+ */
+export function BuildIndexRows({ index_id, ref_id }: { index_id: string; ref_id: string }) {
+    return (
+        <>
+            <JobArgsRow title="Reference">
+                <Link to={`/refs/${ref_id}`}>{ref_id}</Link>
+            </JobArgsRow>
+            <JobArgsRow title="Index">
+                <Link to={`/refs/${ref_id}/indexes/${index_id}`}>{index_id}</Link>
+            </JobArgsRow>
+        </>
+    );
+}
+
+/**
+ * Rows showing important arguments when running an "create_sample" workflow.
+ *
+ * @param args - The complete list of arguments used to run the job
+ * @returns Table rows containing the arguments of the job
+ */
+export function CreateSampleRows({ sample_id }: { sample_id: string }) {
+    return (
+        <JobArgsRow title="Sample">
+            <Link to={`/samples/${sample_id}`}>{sample_id}</Link>
+        </JobArgsRow>
+    );
+}
+
+/**
+ * Rows showing important arguments when running an "create_subtraction" workflow.
+ *
+ * @param args - The complete list of arguments used to run the job
+ * @returns Table rows containing the arguments of the job
+ */
+export function CreateSubtractionRows({ subtraction_id }: { subtraction_id: string }) {
+    return (
+        <JobArgsRow title="Subtraction">
+            <Link to={`/subtractions/${subtraction_id}`}>{subtraction_id}</Link>
+        </JobArgsRow>
+    );
+}
+
+/**
+ * Rows showing important arguments when running an "update_sample" workflow.
+ *
+ * @param args - The complete list of arguments used to run the job
+ * @returns Table rows containing the arguments of the job
+ */
+export function UpdateSampleRows({ sample_id }: { sample_id: string }) {
```

Comment:
I think this may be a relic of a job that no longer exists...

Reply:
I was wondering... I hadn't heard of this job before. Any chance of it still showing up if the user goes back far enough?

Reply:
It will still be handled by your changes with no loss in functionality. Since job records could be deleted when Update Sample was around, they are probably mostly gone anyway.